packages/hardhat-plugin-viem/src/viem-ignition-helper.ts

```diff
@@ -49,54 +57,29 @@ export class IgnitionHelper {
       ContractNameT,
       IgnitionModuleResultsT
     >,
-    {
-      parameters = {},
-      config: perDeployConfig = {},
-    }: {
+    options?: {
       parameters?: DeploymentParameters;
       config?: Partial<DeployConfig>;
-    } = {
-      parameters: {},
-      config: {},
     }
   ): Promise<
     IgnitionModuleResultsTToViemContracts<ContractNameT, IgnitionModuleResultsT>
   > {
-    const accounts = (await this._hre.network.provider.request({
-      method: "eth_accounts",
-    })) as string[];
-
-    const artifactResolver = new HardhatArtifactResolver(this._hre);
-
-    const resolvedConfig: Partial<DeployConfig> = {
-      ...this._config,
-      ...perDeployConfig,
-    };
+    const successfulDeploymentResult =
+      await this._hardhatIgnitionDeployer.deploy(ignitionModule, options);
 
-    const result = await deploy({
-      config: resolvedConfig,
-      provider: this._provider,
-      deploymentDir: this._deploymentDir,
-      artifactResolver,
+    return ViemIgnitionHelper._toViemContracts(
+      this._hre,
       ignitionModule,
-      deploymentParameters: parameters,
-      accounts,
-    });
-
-    if (result.type !== DeploymentResultType.SUCCESSFUL_DEPLOYMENT) {
-      const message = errorDeploymentResultToExceptionMessage(result);
-
-      throw new HardhatPluginError("hardhat-ignition-viem", message);
-    }
-
-    return this._toViemContracts(ignitionModule, result);
+      successfulDeploymentResult
+    );
   }
 
-  private async _toViemContracts<
+  private static async _toViemContracts<
```

Comment:
What's the reasoning behind making this static but not the ethers equivalent functions?

Reply:
Agreed. I will bring the ethers version inline.